cookbook/misc/add_param_to_model_cost_map.py

```diff
@@ -0,0 +1,192 @@
+import json
+import os
+import sys
```

Comment:
storing for later reference - this is a script to programatically insert in litellm model cost map - it's not used for anything related to nova json mode